packages/shared/src/components/cards/v1/Card.tsx

```diff
@@ -45,11 +45,11 @@ export const CardContent = classed('div', 'flex flex-col mobileXL:flex-row');
 
 export const CardImage = classed(
   Image,
-  'rounded-12 h-50 mobileXL:h-28 mobileXL:w-40 mobileXXL:h-auto mobileXXL:w-56',
+  'rounded-12 max-h-[calc(100vw-34px)] mobileXL:aspect-auto mobileXL:max-h-40 mobileXL:w-40 mobileXXL:max-h-56 mobileXXL:w-56',
```

Comment:
The `calc(100vw-34px)` works on mobile screens, because we render the feed full width of the viewport, the padding is 16px on each side and the feed has a 1px border on each side as well.

On bigger screens, with the image being shown on the right column, the max-height is set to a fixed value instead.